apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -246,6 +246,11 @@ export default class SetupChecklist extends Component {
               stepStatus={this.state[STATUS_BOARD_PLUG]}
               stepName="Board plugged in"
             >
+              {this.state.caughtError && this.state.caughtError.reason &&
+                <pre>
+                  {this.state.caughtError.reason}
+                </pre>
+              }
```

Comment:
This `ValidationStep` body only appears if the step has failed - in this case I'm conditionally adding some additional detail to that failure.